lib/astarte/client/appengine/devices.ex

```diff
@@ -17,20 +17,53 @@
 #
 
 defmodule Astarte.Client.AppEngine.Devices do
+  @moduledoc """
+  Module to interact with devices, send and receive data.
+  """
+  @moduledoc since: "0.1.0"
+
   alias Astarte.Client.{APIError, AppEngine}
   alias Astarte.Client.AppEngine.Pagination
 
   @doc """
+  Returns the list of all devices.
+
+  By default the device ID string is returned for each already registered device.
+  The complete device status can be optionally retrieved rather than device ID string using details option.
```

Comment:
-> The complete device status can optionally be retrieved by passing the `details: true` option.